packages/react-dev-utils/package.json

```diff
@@ -66,7 +66,7 @@
     "global-modules": "2.0.0",
     "globby": "11.0.1",
     "gzip-size": "5.1.1",
-    "immer": "8.0.1",
+    "immer": "9.0.6",
```

Comment:
Changing this file but not installing a new version of `react-dev-utils` to BRS, the security issues will still be there. The actions here just remove security issues for the repository, it has nothing to do with BRS package.